fingertip/plugins/script.py

```diff
@@ -1,20 +1,36 @@
 # Licensed under GNU General Public License v3 or later, see COPYING.
 # Copyright (c) 2019 Red Hat, Inc., see CONTRIBUTORS.
 
+import fingertip
 
-def run(m, scriptpath, transient=False, no_unseal=False):
+
+# fingertip ... + script.run script  -  no cache, never persist, always rerun
+# fingertip ... + script.run script + ssh  -  cache just for ssh, never reuse
```

Comment:
what do you mean by "cache just for ssh"?

Reply:
just for this invocation (so that only this particular + ssh will be able to reuse it). will try to rephrase and update.